projects/utils-ogc/tsconfig.lib.json

```diff
@@ -1,8 +1,9 @@
 /* To learn more about this file see: https://angular.io/config/tsconfig. */
 {
-  "extends": "../../tsconfig.base.json",
+  "extends": "../../tsconfig.json",
   "compilerOptions": {
```

Comment:
```
"types": [
      "node"
    ]
```
vscode is showing errors here, do you have the same @MichaelLangbein?
Also I think it would be good to add this to `tsconfig.base.json` so all other libs can use it.

Reply:
Hi @boeckMt !

No, in the current master branch I'm not seeing any errors - neither in the current branch `update-angular`. 

Just for context, those `types: [node]` were added to allow the `utils` library to be used in node-js as well as in the browser. Here's an example (so far the only example there is) of our code making use of node-specific functionality:

utils-ogc/src/lib/wps/wps200/wps_marshaller_200.ts
```typescript
     if (typeof module !== 'undefined' && module.exports) { // node
          data.content.map(c => new Buffer(c, 'base64').toString('ascii'));
        } else { // browser
          data.content.map((c: any) => atob(c));
        }
```

Sure we can add those to the `tsconfig.base`! On one hand, one might argue that only the `utils*` projects might ever be run on node, but on the other hand, simply sharing this type-information automatically between all projects seems preferable over adding it manually to only the `utils*` projects. So I'm all for it.

Reply:
Oh sorry this was my fault, I commented on the wrong PR...
It should be on https://github.com/dlr-eoc/ukis-frontend-libraries/pull/76

Reply:
Not a problem at all. Has been implemented and merged with [PR 76](https://github.com/dlr-eoc/ukis-frontend-libraries/pull/76). Thanks for the heads up!